### ISSUE TITLE
Rocket.Chat webinterface only renders a three dots animation

### ISSUE THREAD

**Issue by mirabilos** [RocketChat/Rocket.Chat#32913]
### Description:

![Untitled](https://github.com/user-attachments/assets/f03235ee-936c-442f-8c92-8cc6a712fb3c)

### Steps to reproduce:

1. Go to the instance of my employer

### Expected behavior:

It should work.

### Actual behavior:

It doesn’t. Furthermore, the F12 console shows:

```
Uncaught TypeError: navigator.serviceWorker is undefined
    ts useDownloadFromServiceWorker.ts:10
    g c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    require c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    o c3d039dad72c15e51dd475aad3adf42252033eb8.js:5
    tsx AttachmentEncryptedDownload.tsx:2
    g c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    require c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    o c3d039dad72c15e51dd475aad3adf42252033eb8.js:5
    tsx AttachmentDownload.tsx:2
    g c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    require c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    o c3d039dad72c15e51dd475aad3adf42252033eb8.js:5
    tsx MessageCollapsible.tsx:1
    g c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    require c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    o c3d039dad72c15e51dd475aad3adf42252033eb8.js:5
    tsx AudioAttachment.tsx:2
    g c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    require c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    o c3d039dad72c15e51dd475aad3adf42252033eb8.js:5
    tsx FileAttachment.tsx:1
    g c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    require c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    o c3d039dad72c15e51dd475aad3adf42252033eb8.js:5
    tsx AttachmentsItem.tsx:2
    g c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    require c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    o c3d039dad72c15e51dd475aad3adf42252033eb8.js:5
    tsx Attachments.tsx:3
    g c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    require c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    o c3d039dad72c15e51dd475aad3adf42252033eb8.js:5
    tsx QuoteAttachment.tsx:2
    g c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    require c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    o c3d039dad72c15e51dd475aad3adf42252033eb8.js:5
    tsx ForwardMessageModal.tsx:2
    g c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    require c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    o c3d039dad72c15e51dd475aad3adf42252033eb8.js:5
    ts messageActionDefault.ts:2
    g c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    require c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    o c3d039dad72c15e51dd475aad3adf42252033eb8.js:5
    ts index.ts:1
    g c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    require c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    o c3d039dad72c15e51dd475aad3adf42252033eb8.js:5
    ts accounts.ts:1
    g c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    require c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    o c3d039dad72c15e51dd475aad3adf42252033eb8.js:5
    ts main.ts:1
    g c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    require c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    t c3d039dad72c15e51dd475aad3adf42252033eb8.js:3
    <anonymous> transforms.ts:24
useDownloadFromServiceWorker.ts:10
There is no route for the path: /
```

### Server Setup Information:

- Version of Rocket.Chat Server: 
- License Type:
- Number of Users: 
- Operating System: 
- Deployment Method: 
- Number of Running Instances: 
- DB Replicaset Oplog: 
- NodeJS Version: 
- MongoDB Version:

### Client Setup Information

- Desktop App or Browser Version: Firefox ESR 115.13
- Operating System: Debian bullseye/amd64

**Comment by MartinSchoeler** [comment]
Hello @mirabilos this issue is likely being caused by the lack of support for service workers on firefox anonymous window

This was fixed in https://github.com/RocketChat/Rocket.Chat/pull/32807 and it will be shipped on version 6.11

A work around for now is to use another browser or the desktop app, thanks!

**Comment by mirabilos** [comment]
Or a manual, deliberate deactivation of service workers, yes, that would do it if you accidentally relied on those.
